protocol/messenger_pin_messages.go

```diff
@@ -0,0 +1,89 @@
+package protocol
+
+import (
+	"context"
+	"errors"
+
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+// SendPinMessage sends the PinMessage to the corresponding chat
+func (m *Messenger) SendPinMessage(ctx context.Context, message *common.PinMessage) (*MessengerResponse, error) {
+	m.mutex.Lock()
+	defer m.mutex.Unlock()
+	return m.sendPinMessage(ctx, message)
+}
+
+func (m *Messenger) sendPinMessage(ctx context.Context, message *common.PinMessage) (*MessengerResponse, error) {
+	var response MessengerResponse
+
+	// A valid added chat is required.
+	chat, ok := m.allChats[message.ChatId]
+	if !ok {
+		return nil, errors.New("Chat not found")
```

Comment:
Tiny little thing, go convention is not to start an error message with a capital letter.

```suggestion
		return nil, errors.New("chat not found")
```